recipes/ffmpeg/all/conanfile.py

```diff
@@ -902,11 +908,20 @@ def _add_component(name, dependencies):
             if Version(self.version) >= "5.0" and is_apple_os(self):
                 avfilter.frameworks.append("Metal")
 
+        if self.options.get_safe("with_libdrm"):
+            avutil.requires.append("libdrm::libdrm_libdrm")
         if self.options.get_safe("with_vaapi"):
-            avutil.requires.extend(["vaapi::vaapi", "xorg::x11"])
+            avutil.requires.append("libva::libva_")
+            if self.options.get_safe("with_libdrm"):
+                avutil.requires.append("libva::libva-drm")
+            if self.options.get_safe("with_xcb"):
+                avutil.requires.extend(["libva::libva-x11", "xorg::x11"])
 
         if self.options.get_safe("with_vdpau"):
-            avutil.requires.append("vdpau::vdpau")
+            avutil.requires.append("libvdpau::libvdpau")
+
+        if self.options.with_ssl == "openssl":
```

Comment:
I can confirm this fix working, I tested locally on Linux. I would ask to split this PR. Those extra dependencies will require extra steps to be validated, and replacing packages is even more fragile as ffmpeg is a critical package.

Reply:
I've reverted those changes to use the `libva` and `libvdpau`.